includes/compatibility/civicrm.wpml.php

```diff
@@ -100,4 +106,264 @@ public function locale_filter($locale) {
 
   }
 
+ /**
```

Comment:
I think you have 1 space here instead of 2, so the style tests are failing.

Reply:
Super funny because I just changed them to be shorter because it was complaining but I think I misunderstood the problem. Thanks for the guidance.